### PR TITLE
[SES-108] Deep links

### PR DESCRIPTION
**Pull request by YasielCabrera**
# Ticket
https://trello.com/c/trpIoYYf/108-implement-seo-elements-v-10

# Description
The deep links using anchor in the URL (#id) are changed when the user clicks on the target element but the rest of the work is done using TS to prevent default behavior and  wrong placement of the scroll

# Actions
- removed unused URL params
- implement deep links for the transparency tab
- implement deep links for transparency actuals and transparency forecast tabs

### CODE REVIEW

File: src/stories/containers/transparency-report/transparency-report.tsx [hunk 1]
```diff
@@ -56,24 +57,64 @@ export const TransparencyReport = ({
...
+      query: {
+        ...router.query,
+        viewMonth
+      },
+    }, undefined, {
+      shallow: true,
+    });
+  };
+
+  const handlePreviousMonth = () => {
```
Comment (alejandrocabriales) [inline src/stories/containers/transparency-report/transparency-report.tsx:106]: What do you think about use useCallback hook for this functions??

File: src/stories/containers/transparency-report/transparency-report.tsx [hunk 2]
```diff
@@ -56,24 +57,64 @@ export const TransparencyReport = ({
...
+    const month = currentMonth.minus({ month: 1 });
+    replaceViewMonthRoute(month.toFormat('LLLyyyy'));
+    setCurrentMonth(month);
+  };
+
+  const handleNextMonth = () => {
```
Comment (alejandrocabriales) [inline src/stories/containers/transparency-report/transparency-report.tsx:112]: What do you think about use useCallback hook for this functions??

File: src/stories/containers/transparency-report/transparency-report.tsx (hunk 1 above)
Comment (YasielCabrera) [reply src/stories/containers/transparency-report/transparency-report.tsx]: It's a good idea, I changed it

File: src/stories/containers/transparency-report/transparency-report.tsx (hunk 2 above)
Comment (YasielCabrera) [reply src/stories/containers/transparency-report/transparency-report.tsx]: It's a good idea, I changed it

**Comment by alejandrocabriales** [comment]
LGTM

**Pull request merged**